rust/extractor/src/translate/base.rs

```diff
@@ -363,10 +361,10 @@ impl<'a> Translator<'a> {
             .as_ref()
             .and_then(|s| s.expand_macro_call(mcall))
         {
-            self.emit_macro_expansion_parse_errors(mcall, &expanded);
+            self.emit_macro_expansion_parse_errors(mcall, &expanded.value);
```

Comment:
Does `expanded` also have an `err` field? If so, let log it, hopefully it'll provide some useful information of why an expansion failed?